lib/builders/observability-builder.ts

```diff
@@ -0,0 +1,52 @@
+import * as blueprints from '../../lib';
+import * as utils from "../utils";
+import { NestedStack, NestedStackProps } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+
+export class ObservabilityBuilder extends blueprints.BlueprintBuilder {
+
+    public addNewClusterObservabilityBuilderAddOns(): ObservabilityBuilder {
+        return this.addOns(
+            new blueprints.addons.VpcCniAddOn(),
+            new blueprints.addons.CoreDnsAddOn(),
+            new blueprints.addons.MetricsServerAddOn(),
+            new blueprints.addons.PrometheusNodeExporterAddOn(),
+            new blueprints.addons.KubeStateMetricsAddOn());
+    }
+
+    public addExistingClusterObservabilityBuilderAddOns(): ObservabilityBuilder {
+        return this.addOns(
+            new blueprints.addons.AwsLoadBalancerControllerAddOn(),
```

Comment:
Why Cert manager and AWS Load balancer controller not included in the new cluster addons but are included in the existing cluster? what relevance do they have for existing cluster, for example what if the existing cluster already has this software?

Reply:
The relevance is we have bunch of addons to new cluster addons but we have nothing in existing cluster addons. So we cant keep the existing cluster addons array empty. I know this will impact if the existing cluster already has ALB and Cert manager but not sure how to approach this. Any thoughts if we can leave addons array empty in existing cluster?

Reply:
So the new cluster add-ons will just not work on the existing cluster, is that what you are saying?